nextflow.config

```diff
@@ -90,6 +88,22 @@ profiles {
         includeConfig 'conf/codon_slurm.config'
         includeConfig 'conf/codon_dbs.config'
     }
+
+    local {
+        includeConfig 'conf/local.config'
+        includeConfig 'conf/local_dbs.config'
+    }
+}
+
+manifest {
+    name            = 'ebi-metagenomics/amplicon-pipeline'
+    author          = """Microbiome Informatics Team - EMBL-EBI"""
+    homePage        = 'https://github.com/EBI-Metagenomics/amplicon-pipeline'
+    description     = """Microbiome Informatics metagenomes amplicon analysis pipeline"""
+    mainScript      = 'main.nf'
+    nextflowVersion = '!>=23.04.0'
+    version         = '1.0.0'
```

Comment:
Is this the right version?

Reply:
the pipeline has never really had a real version yet... might as well start with 1.0.0 I think?

Reply:
eah, I wasn't sure about the version. 1.0.0 seems like a good choice; it has had plenty of testing up to this point.